ring_buffer/ring_buffer.py

```diff
@@ -1,11 +1,64 @@
 class RingBuffer:
-  def __init__(self, capacity):
-    self.capacity = capacity
-    self.current = 0
-    self.storage = [None]*capacity
+    def __init__(self, capacity):
+        self.capacity = capacity
+        self.current = 0
+        self.storage = [None]*capacity
 
-  def append(self, item):
-    pass
+    def append(self, item):
+         
+        if self.current >= self.capacity:
+          self.current = 0
+ 
+        self.storage[self.current] = item
+        self.current += 1
 
-  def get(self):
-    pass
\ No newline at end of file
+        # return self.storage
+
+    def get(self):
+        getStorage = []
+
+        for i in range(self.capacity):
+            if self.storage[i] != None:
+                getStorage.append(self.storage[i])
+
+        return getStorage
+
+
+        
+
+
+"""
+Test in Terminal:
+cd in folder
+$ python3
+from ring_buffer import RingBuffer
+temp = RingBuffer(3)
+temp.get()
+temp.append("A")
+temp.append("B")
+temp.get()
+temp.append("EE")
+temp.get()
+"""
+
+
```

Comment:
I like how you planned everything first, pseudo-code is definitely s good skill to have.